Ang-Li/abstracts_bios.md

```diff
@@ -0,0 +1,22 @@
+# Abstracts and bios
+
+This is a file of abstracts and applications for events I attend in 2019.
+
+### Cambridge Networks Day
+
+**Website**: https://www.cnn.group.cam.ac.uk/cambridge-networks-day
+
+**Date**: 29 August 2019
+
+**Application**: 
+I hope to attend the Cambridge network day, as I'm super interested at network science.
+As a Ph.D. student focusing on neuroimaging and data science, I already have some background knowledge of network and its application 
+in neuroimaging. Our brain, just like the other complex systems in the real world, is a small world network, and the topological structure 
```

Comment:
I know I'm really annoying about this - please can you put each new sentence on a new line?

Reply:
Of course, have fixed this point

Reply:
Not all of the sentences are on new lines. Please read through carefully and fix all of them in this paragraph.